styles/pages/Sign.styled.ts

```diff
@@ -41,14 +41,15 @@ export const SignBox = styled.div`
     color: var(--main-color);
     position: relative;
     top: -4px;
+    border-bottom: 1px solid var(--main-color);
   }
 `;
 
 export const EmailBox = styled.div`
   margin-top: 24px;
 `;
 
-export const PassWrodBox = styled.div`
+export const PassWordBox = styled.div`
```

Comment:
password 는 한 단어이므로 중간에 대문자 대신 전체 소문자로 작성해야해요 
```suggestion
export const PasswordBox = styled.div`
```